unattended_installer/install_functions/filebeat.sh

```diff
@@ -54,6 +54,8 @@ function filebeat_copyCertificates() {
             eval "tar -xf ${tar_file} -C ${filebeat_cert_path} wazuh-install-files/root-ca.pem --strip-components 1 ${debug}"
             eval "rm -rf ${filebeat_cert_path}/wazuh-install-files/ ${debug}"
         fi
+        eval "chmod 500 ${dashboard_cert_path} ${debug}"
+        eval "chmod 400 ${dashboard_cert_path}/* ${debug}"
```

Comment:
```suggestion
        eval "chmod 400 ${filebeat_cert_path}/* ${debug}"
```

The name of the variable is incorrect

Reply:
Done [ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41](https://github.com/wazuh/wazuh-packages/pull/1415/commits/ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41)